### DISCUSSION TITLE
Column Name: "Unit Price", "D/C"

### DISCUSSION THREAD

**Discussion by hkpeaks** [duckdb/duckdb#7605] [Ideas]
The popularity of dataframe software such as Polars and Pandas empower to support column name having white space and "/".
Below column name is very common to accounting dataset. Today I use DuckDB distinct function, fail to recognize these 2 kinds of column names.

Date, Ledger, Account, Part No, Project, Contact, Unit Code, Quantity, Unit Price, D/C, Currency, Exchange Rate, Original Amount, Base Amount

**Comment by Tishj** [comment]
Have you tried quoting the column names?
"Date", "Ledger", "Part No", etc..

**Comment by hkpeaks** [comment]
I am a programmer. Since there are a comma separator to identify each column name, my practice do not use "      " to handle whitespace or other special character.

Now I use it can get the results.

SELECT DISTINCT Ledger, Account, PartNo, Project, "D/C", "Unit Code", Contact, Currency FROM 'Input/1000MillionRows.parquet';

**Reply by Alex-Monahan** [reply]
The reason you need the quotes is twofold: 
1. SQL aliases are separated by spaces (so `select A B` is actually selecting the data of column A and renaming to B.
2. The forward slash is used to mean division

Double quotes clear up your intent!

**Reply by hkpeaks** [reply]
I am not experience in using SQL statement as I have my own designed ETL expression. A = B is my column rename method.
